packages/components/e2e/component-objects/src/main/java/org/talend/component/Application.java

```diff
@@ -13,6 +13,7 @@
 
     public WebDriver driver;
 
+    // TODO (breaking change) : Rename theses variable in order to respect the Java Convention, and set them private in order to use getter
     public SidePanel SidePanel;
```

Comment:
Why didn't you made this change ?

Reply:
Because the variables names doesn't respect the classic Java checkstyle, and i think it is a bad practice to make them public. We should use getter. But this is a breaking change, and i'am not sure that these reasons are enough significant to deserve the breaking change. So i wanted to open the discussion.